tests/console/bash.pm

```diff
@@ -0,0 +1,80 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Bash regression test (https://progress.opensuse.org/issues/50747)
+# Maintainer: Nalmpantis Orestis (onalmpantis)
+ 
+use base "consoletest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle'; 
+
+sub run {
+    # Declaring repo and database variables that change depending on the product tested	
+    my $repo="";
+    my $database="";
+    if (is_sle('=12-sp1')) {
+	    $repo = "http://download.suse.de/ibs/QA:/SLE12SP1/update/QA:SLE12SP1.repo";
+	    $database = "failed_tests_database_sle12-sp1";
+    } 
+    if (is_sle('=12-sp2')) {
+	    $repo = "http://download.suse.de/ibs/QA:/SLE12SP2/update/QA:SLE12SP2.repo";
+	    $database = "failed_tests_database_sle12-sp2";
+    } 
+    if (is_sle('=12-sp3')) {
+	    $repo = "http://download.suse.de/ibs/QA:/SLE12SP3/update/QA:SLE12SP3.repo";
+	    $database = "failed_tests_database_sle12-sp3";
+    } 
+    if (is_sle('=12-sp4')) {
+	    $repo = "http://download.suse.de/ibs/QA:/SLE12SP4/update/QA:SLE12SP4.repo";
+	    $database = "failed_tests_database_sle12-sp4";
+    } 
+    if (is_sle('=15')) {
+	    $repo = "http://download.suse.de/ibs/QA:/SLE15/update/QA:SLE15.repo";
+	    $database = "failed_tests_database_sle15";
+    }
+    if (is_sle('=15-sp1')) {
+            $repo = "http://download.suse.de/ibs/QA:/SLE15SP1/update/QA:SLE15SP1.repo";
+            $database = "failed_tests_database_sle15-sp1";
+    }
+ 
+    select_console "root-console";
+    # Adding the testsuite repository
+    assert_script_run "zypper ar --no-gpgcheck $repo";
+    # Installing the testsuite
+    assert_script_run "zypper --non-interactive --no-gpg-checks in qa_test_bash";
+    # Running the testsuite
+    assert_script_run "/usr/share/qa/tools/test_bash-run", timeout => 1200;	    
+    # Locating the folder with testsuite results
+    assert_script_run "cd /var/log/qa/ctcs2/";
+    # Making sure the newest created folder is picked
+    assert_script_run 'cd `ls -th|head -n 1`';
```

Comment:
I will check this

Reply:
The -h is for using human eye friendly expression of file/directory sizes. What has no meaning  with only the -t.
But at the same time, it does not hurt anything.

Reply:
Sure, I just wanted to know, if author meant something special with that, because using `-h` without `-l` is, well.. weird and useless as there are no directory sizes.